docs/_includes/date-picker.html

```diff
@@ -0,0 +1,144 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+    <p>Arrow keys should navigate between dates, but this is an implementation detail that is not yet included in skin.</p>
+
+    <h3>Single Pane</h3>
+
+    <div class="date-picker" aria-label="date picker">
```

Comment:
Aria label is redundant on elements with no implicit or explicit role. One option to explore _could_ be `role=group aria-roledescription=date picker`, but let's just remove aria-label for now and we can explore our options later.

Reply:
I take it back, aria-label is now valid on all elements, however I just don't feel like we need it here.